clients/rospy/src/rospy/topics.py

```diff
@@ -995,6 +995,9 @@ def add_connection(self, c):
         """
         if self.queue_size is not None:
             c = QueuedConnection(c, self.queue_size)
+        else:
+            import warnings
+            warnings.warn("Adding non-queued connection for topic %s to [%s:%s]" % (self.resolved_name, c.remote_endpoint[0], c.remote_endpoint[1]))
```

Comment:
Can't use `_logger.warn` before ros is fully initialized or messages will be swallowed.

Reply:
Any idea what logger this uses? The `rospy.topics` logger is set to FATAL by default so we might not see it.
``` 
 -                                                                                                                                                                                                         
    name: "rospy.topics"                                                                                                                                                                                    
    level: "FATAL"

Reply:
I'm not sure. After normal rospy logging didn't work during testing I copied what I saw elsewhere in the initialization code: https://github.com/locusrobotics/ros_comm/blob/a30195569b4b5029be43eadfb4553462b1af9257/clients/rospy/src/rospy/topics.py#L148

You might be right and this was just a case of me not setting the logging level before running the node 🤦